modules/1plusXRtdProvider.js

```diff
@@ -159,12 +159,7 @@ export const buildOrtb2Updates = ({ segments = [], topics = [] }) => {
     segment: topics.map((topicId) => ({ id: topicId })),
     ext: { segtax: segtaxes.CONTENT }
   }
-  // Currently appnexus bidAdapter doesn't support topics in `site.content.data.segment`
-  // Therefore, writing them in `site.keywords` until it's supported
-  // Other bidAdapters do fine with `site.content.data.segment`
-  const siteKeywords = topics.map(topic => `1plusX=${topic}`).join(',');
```

Comment:
Can we clarify with 1plusX how they passed their segment keywords previously?  It seem like here it was with a keyname like `1plusX` but the new logic has `1plusx`.  We should follow the same formatting as it was before.

Reply:
fixed!